app/lib/analyzeResponse.js

```diff
@@ -1,4 +1,4 @@
-export const analyzeResponse = (data) => {
+const analyzeResponse = (data) => {
   let scores = data[0].scores;
   let emotions = Object.keys(scores);
   let sortedEmotions = emotions.sort((a, b) => {
```

Comment:
these can/should all be `const` instead of `let` since you're not reassigning them.